### DISCUSSION TITLE
Zoom %?

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#651] [Q&A]
Is it just me, or is the zoom percentage strange?
When I load a map, I'm at 100%, when I zoom *out* it goes up to 1550%, and when I zoom *in* it goes down to 50%.

Shouldn't 50% be half of the regular scale? Right now it's double, so I'd be inclined to call that 200%.

Playing on Android, latest unstable build.

**Comment by crudelios** [comment]
Hey, I know you're technically correct, but doing it the proper way would result in values being always very low.

I mean, of instead of saying "zoom 1500%" it said "zoom 6%" it wouldn't look as good.

**Reply by catalin-duta** [reply]
So what if instead of Zoom 50% (actually 200%) it was worded as Scale 2:1 and Zoom 1500% becomes Scale 1:150, or something along those lines?
